lib/api/2.0/users.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var _ = injector.get('_'); // jshint ignore:line
+var configuration = injector.get('Services.Configuration');
+var waterline = injector.get('Services.Waterline');
+var Errors = injector.get('Errors');
+var aclService = injector.get('Acl.Services');
+var localUserException = configuration.get('enableLocalHostException', true);
+
+var addUser = controller({success: 201}, function(req, res) {
+    var userObj = _.pick(req.swagger.params.body.value, ['username', 'password', 'role']);
+    userObj.role = userObj.role || 'ReadOnly';  //TODO: move to Constants when Role code is added
+
+    return waterline.localusers.find({}).then(function(users) {
+        if(!users.length && localUserException && res.locals.ipAddress === '127.0.0.1') {
+            // Only when there are no users, and the remote is a local connection, and we 
```

Comment:
@zyoung51 I'm thinking we may want to force this user into the admin role or return an error if the posted user isn't admin.